labs/jack-in/src/config.rs

```diff
@@ -37,6 +37,7 @@ pub struct Opt {
 }
 
 #[derive(Debug, Clone, ValueEnum)]
+#[value(rename_all = "PascalCase")]
```

Comment:
I think we shouldn't actually do this, lowercase seems more reasonable for the full-sync-mode options. wdyt @gnunicorn?

Reply:
I did the same in my [PR](https://github.com/matrix-org/matrix-rust-sdk/pull/1278/files#diff-99db98c3907c1fb2ed7acb0ff35508673e555131315a69921cfe16b6346a2d4dR39), I added this because it was already in PascalCase in the old cli with structopt (because it was using clap's [arg_enum!](https://docs.rs/clap/2.34.0/clap/macro.arg_enum.html) without [case_insensitive()](https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.case_insensitive))

But yeah, lowercase seems a lot better

Reply:
Yeah i also used PascalCase because it already was like that in structopt, but i can quickly change it to lowercase if thats better.

Reply:
Change is pushed